crates/cansat-stm32f4/src/startup.rs

```diff
@@ -144,6 +143,22 @@ fn init_sd_logger(spi: Spi2, cs: Cs2, statik: &'static mut Statik) -> Result<SdL
     Ok(logger)
 }
 
+fn init_lora(serial6: Serial6) -> Result<Lora, LoraError> {
+    defmt::info!("Initializing LORA");
+
+    let mut lora = Lora::new(serial6);
+    let mut resp_buffer: [u8; 64] = [0; 64];
+    const COMMANDS: [&str; 3] = ["AT\r\n", "AT+MODE=TEST\r\n", "AT+UART=TIMEOUT,4000\r\n"];
+
+    for cmd in COMMANDS {
+        let resp_len = lora.send(cmd.as_bytes(), &mut resp_buffer)?;
+        cansat_lora::parse::response(&resp_buffer).map_err(|_| LoraError::Parse)?;
```

Comment:
Implement `From<cansat_lora::parse::Error>` for `cansat_lora::Error::Parse`. Then you can remove the manual error transformation, `?` operator will do it for you
```suggestion
        cansat_lora::parse::response(&resp_buffer)?;
```